services/supervisor/src/api.rs

```diff
@@ -50,8 +48,8 @@ pub trait PublicApi {
     type Error: Fail;
     /// Returns an actual consensus configuration of the blockchain.
     fn consensus_config(&self) -> Result<ConsensusConfig, Self::Error>;
-    /// Returns an pending propose config change.
-    fn config_proposal(&self) -> Result<Option<ConfigProposalWithHash>, Self::Error>;
+    /// Returns pending proposals config change.
```

Comment:
It seems like some preposition is missed between `proposals` and `config change`. According to [google](https://english.stackexchange.com/questions/304604/the-proposal-for-or-the-proposal-of) and several other sites, `for` is needed here.